app/App.js

```diff
@@ -47,6 +48,7 @@ class App extends React.Component<Props, State> {
       <>
         <GlobalStyle />
         <Provider store={store}>
+          <QuitDialog />
```

Comment:
move it to Main and increase css dimensions (top, bottom, left, right) to cover the whole screen

Reply:
Ok on the dimensions, but if it's moved to Main it would need to be in Auth as well to enable quitting from there as well.

Reply:
When user is logged out there should be no notification cause there is no control of node

Reply:
True, but the event is intercepted at main.dev. That means we still need to listen to 'REQUEST_CLOSE' in auth in order to complete the app quit, no? On app close in main dev we're just sending 'REQUEST_CLOSE'.

Reply:
simply check location before changing visibility and in case of auth close immediately